source/docs/advanced/performance/overview.md

```diff
@@ -5,11 +5,11 @@ title: Overview
 
 In this section, you will learn advanced information about Performance in Front-Commerce.
 
-* [Preloading Routes' data](./preloading-routes.html)
```

Comment:
out of curiosity why are u changing the * to a -

Reply:
This was an automated formatting:

> I noticed that some code was reformatted during git and/or editor hooks. I've reverted as many changes as possible, but some were a bit more tedious.